dali/base/dadfs.cpp

```diff
@@ -8496,6 +8496,8 @@ class CFileScanFilterContainer : public CInterface
 {
     StringAttr filterBuf; //Hold original filter string just in case
     StringAttr wildNameFilter;
+    unsigned pageFrom;
+    unsigned pageTo;
```

Comment:
I don't think a range make sense, vs the 'max' which was previously discussed.
If, from was >0 then it would imply the server (Dali) was being asked for to start at some point N , when scanning, with there being every chance that the results had been changed since the last request.

This is unlike the existing mechanism where the result set is cached and startOffset -> startOffset+pageSize is used to slice through it.